src/Modules/Header/HeaderComponent.js

```diff
@@ -32,7 +32,7 @@ const HeaderComponent = (props) => {
         setNavbarShrink("");
       }
     };
-  }, [localStorage.getItem("isLoggedIn"), isLoggedIn]);
+  }, [localIsLoggedIn, isLoggedIn]);
```

Comment:
I'll admit this is a little weird coming from the react team to specify that this should be it's own variable but that's what they suggest.